internal/backend/temporalclient/config.go

```diff
@@ -62,10 +62,19 @@ var (
 				Encryption: DataConverterEncryptionConfig{
 					Encrypt: true,
 				},
+				LargePayload: LargePayloadConfig{
```

Comment:
Optional: until we have D3 configured, why not specify in prod one of the other storage options, temporarily, just in case?

Reply:
changed